web/pom.xml

```diff
@@ -1162,6 +1162,42 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>docker</id>
+      <properties>
+        <dockerImageName>georchestra/${project.parent.artifactId}</dockerImageName>
```

Comment:
It will require an extra docker repository, and I think that the version number (gn2 or 3) should be carried by the docker tag. For me it is relevant to have both versions nested in the same repository.